poetro_app/lib/features/poetry/presentation/bloc/poetry_bloc/poetry_bloc.dart

```diff
@@ -20,47 +22,59 @@ class PoetryBloc extends Bloc<PoetryEvent, PoetryState> {
 
   final GetRandomPoemSequenceUsecase _getRandomPoemSequenceUsecase;
 
+  final GetPoetryListByKeyword _getPoetryListByKeyword;
+
   PoetryBloc({
     required GetPoetryByCountUsecase getPoetryByCountUsecase,
     required FetchRandomPoemUsecase fetchRandomPoemUsecase,
     required GetRandomPoemSequenceUsecase getRandomPoemSequenceUsecase,
+    required GetPoetryListByKeyword getPoetryListByKeyword,
   })  : _getPoetryByCountUsecase = getPoetryByCountUsecase,
         _fetchRandomPoemUsecase = fetchRandomPoemUsecase,
         _getRandomPoemSequenceUsecase = getRandomPoemSequenceUsecase,
```

Comment:
you separate the class in use cases but use whol in the same bloc, it means you not need any use case